CMakeLists.txt

```diff
@@ -19,18 +25,30 @@ set(VERSION_BUILD ${BUILD_NUMBER})
 # Used for write_basic_package_version_file()
 project(triton VERSION ${VERSION_MAJOR}.${VERSION_MINOR})
 
+
+
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_GENERATOR_TOOLSET STREQUAL "ClangCL")
+    SET(CMAKE_CXX_FLAGS "/EHsc")
```

Comment:
If you can add a comment about what those flags means :)

Reply:
> If you can add a comment about what those flags means :)

If you're using LLVM on windows, clang-cl is recommended over MSVC.

Reply:
> If you can add a comment about what those flags means :)

If you want to use the clang - cl, requires this setting EHsc, prevent errors